README.md

```diff
@@ -121,7 +179,8 @@ java -cp ./dbeam-core/target/dbeam-core-shaded.jar \
   --table=my_table
 ```
 
-- Replace postgres with mysql if you are using MySQL.
+- Replace postgres with mysql if you are using MySQL: `--connectionUrl=jdbc:mysql://google/database?socketFactory=com.google.cloud.sql.mysql.SocketFactory&cloudSqlInstance=project:region:cloudsql-instance&useCursorFetch=true`
```

Comment:
should we have this separate for postgres and mysql?

Reply:
Good point. Trying to avoid many lines for each combination here. Still, updated to make it clearer..